src/jit/codegencommon.cpp

```diff
@@ -9406,9 +9422,6 @@ void                CodeGen::genFnEpilog(BasicBlock* block)
             compiler->unwindBegEpilog();
             unwindStarted = true;
         }
-        // mov FP into SP
-        inst_RV_RV(INS_mov, REG_SPBASE, REG_FPBASE);
-        compiler->unwindSetFrameReg(REG_FPBASE, 0);
```

Comment:
I don't think this unwind code is right, so I didn't copy it for frameType =1 case -- we are not setting fp from sp but sp from fp.
Likewise, we'd like to adjust sp from fp (for other frameTypes), but I'm not sure how we express such things in unwind code. It appears sp adjustment api is only for delta from the current sp. Or don't we care at all in epilog? @BruceForstall Any comment?